### PR TITLE
mapped android support packages to androidX packages

### PR DESCRIPTION
**Pull request by udhay24**
solves #12 
### Gradle Changes:
changed compile time sdk to 28
Gradle version to 3.3.2

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -20,24 +20,24 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation 'com.android.support:appcompat-v7:27.1.1'
-    implementation 'com.android.support:design:27.1.1'
-    implementation 'com.android.support.constraint:constraint-layout:1.1.2'
+    implementation 'androidx.appcompat:appcompat:1.0.0-beta01'
```
Comment (jbeorse) [inline app/build.gradle]: Do we need to use beta versions? Are there stable versions that we can use?

Comment (udhay24) [reply app/build.gradle]: yeah I have made the changes. everything is updated to stable versions

**Review by jbeorse — APPROVED** [review]

**Comment by jbeorse** [comment]
I have approved the changes but you will need to resolve the conflicts before we can merge.

**Comment by udhay24** [comment]
@jbeorse I don't have the access

**Comment by udhay24** [comment]
@jbeorse i have solved the merge conflicts. you can merge them

**Comment by jbeorse** [comment]
Thanks @udhay24. Apologies, but could you actually remove the files in the `.idea` folder as well? In the long run I'd rather not have those files checked in at all since they can be different for each machine. After that I can merge the PR.

**Comment by udhay24** [comment]
@jbeorse i have removed the .idea folder.

**Pull request merged**